PhotoCollection/Model Controller/PhotoController.swift

```diff
@@ -0,0 +1,31 @@
+//
+//  PhotoController.swift
+//  PhotoCollection
+//
+//  Created by Chris Gonzales on 1/30/20.
+//  Copyright © 2020 Chris Gonzales. All rights reserved.
+//
+
+import Foundation
+
+class PhotoController{
+    
+    
+    // MARK: Properties
+    
+    var photos = [Photo]()
+    
+    // MARK: CRUD Methods
+    
+    func createPhoto(photo: Photo){
+        let newPhoto = photo
+        photos.append(newPhoto)
+    }
```

Comment:
When creating a new photo you also want to include the other arguments that make up a photo. In this case it is an image and title. 
EX: `func createPhoto(with imageData: Data, title: String) {`
Then you create a Photo to a constant.
EX: `let newPhoto = Photo(imageData: imageData, title: title)`